main.md

```diff
@@ -1,9 +1,18 @@
 ## GIT CHEATSHEET
 
 ### Common Commands
+`$ git init`
+- It will create a new git repository in the current directory.
+
+`$ git init --bare`
+- it will create a new bare repository (no .git folder, history is stored in the project root).
+
 `$ git clone <link to repo>`
 - Clones repository to local machine.
 
+`$ git clone --bare <link to reo>`
+- Do a bare copy of the repository you want to clone.
```

Comment:
Write a line on how this is different from git clone !

Reply:
Can I add a note at the end of the chapter?

Reply:
I think it would be better to add the difference here itself.